packages/fxa-settings/src/pages/Signin/index.stories.tsx

```diff
@@ -0,0 +1,72 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import Signin, { SigninProps } from '.';
+import AppLayout from '../../components/AppLayout';
+import { RelyingParties } from '../../lib/types';
+import { LocationProvider } from '@reach/router';
+import { Meta } from '@storybook/react';
+import { MOCK_EMAIL, MOCK_SERVICE, MOCK_OTHER_ICON } from './mocks';
+export default {
+  title: 'pages/Signin',
+  component: Signin,
+} as Meta;
+
+// TODO: Add in error and success states when the Alert Bar is implemented
```

Comment:
We might want to take off `when the Alert Bar is implemented` since I think we'll be using a new `Banner` component I've got in my WIP PR.